lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -67,36 +61,77 @@ export class CdnSiteHostingConstruct extends cdk.Construct {
     });
     new cdk.CfnOutput(this, "Bucket", { value: this.s3Bucket.bucketName });
 
+    const defaultSecurityHeaders: cloudfront.ResponseSecurityHeadersBehavior = {
+      // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src
+      contentSecurityPolicy: {
+        contentSecurityPolicy: "default-src 'self';",
+        override: true,
+      },
+      // https://web.dev/security-headers/#xcto
+      contentTypeOptions: { override: true },
+      // https://web.dev/security-headers/#recommended-usages-4
+      frameOptions: {
+        frameOption: cloudfront.HeadersFrameOption.DENY,
+        override: true,
+      },
+      // https://web.dev/referrer-best-practices/#setting-your-referrer-policy:-best-practices
+      referrerPolicy: {
+        referrerPolicy:
+          cloudfront.HeadersReferrerPolicy.STRICT_ORIGIN_WHEN_CROSS_ORIGIN,
+        override: true,
+      },
+      // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#examples
+      strictTransportSecurity: {
+        accessControlMaxAge: Duration.millis(31536000),
```

Comment:
question: what is this number? Could it be a constant or at least a calculation that makes it human-readable?

Reply:
It is 2 years I can make it a constant, I left it as its fairly familiar if you are used to adding a strict transport security policy.

Reply:
Made it a variable.